.clang-tidy

```diff
@@ -35,13 +35,14 @@ cppcoreguidelines-*,
 hicpp-exception-baseclass,
 hicpp-avoid-goto,
 misc-*,
+-misc-confusable-identifiers,
 -misc-const-correctness,
 -misc-include-cleaner,
 -misc-use-anonymous-namespace,
 -misc-unused-parameters,
 -misc-no-recursion,
 -misc-non-private-member-variables-in-classes,
--misc-confusable-identifiers,
+-misc-unused-using-decls,
```

Comment:
Where are all the problematic unused-using-decls?

Reply:
Some headers in ten contain only using statements for used in other cpp files.